_nonmvptools/capetownbudgetproject.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: tool
-title: Cape Town Budget Project
+name: Cape Town Budget Project
```

Comment:
Yes this was a mistake, I meant to keep `name` all lowercase. Thanks for pointing that out